From 3de32d0a4fbbe2d39b3e07f04a162d0fe3a4424c Mon Sep 17 00:00:00 2001
From: Christophe Priouzeau <christophe.priouzeau@st.com>
Date: Mon, 6 Feb 2017 15:46:50 +0100
Subject: [PATCH] ARM Support

Signed-off-by: Christophe Priouzeau <christophe.priouzeau@st.com>
---
 Makefile | 20 ++++++++++++++++++--
 1 file changed, 18 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 1d031c2..cd5cb75 100644
--- a/Makefile
+++ b/Makefile
@@ -15,7 +15,7 @@ EXTRA_CFLAGS += -Wno-unused-label
 EXTRA_CFLAGS += -Wno-unused-parameter
 EXTRA_CFLAGS += -Wno-unused-function
 EXTRA_CFLAGS += -Wno-unused
-EXTRA_CFLAGS += -DCONFIG_CONCURRENT_MODE
+#EXTRA_CFLAGS += -DCONFIG_CONCURRENT_MODE
 
 ccflags-y += -D__CHECK_ENDIAN__
 
@@ -53,7 +53,7 @@ CONFIG_PNO_SUPPORT = n
 CONFIG_PNO_SET_DEBUG = n
 CONFIG_AP_WOWLAN = n
 ###################### Platform Related #######################
-CONFIG_PLATFORM_I386_PC = y
+#CONFIG_PLATFORM_I386_PC = y
 ###############################################################
 
 CONFIG_DRVEXT_MODULE = n
@@ -249,6 +249,22 @@ EXTRA_CFLAGS += -DCONFIG_GPIO_WAKEUP
 endif
 
 
+ifeq ($(CONFIG_SOC_STIH407), y)
+EXTRA_CFLAGS += -DCONFIG_IOCTL_CFG80211
+EXTRA_CFLAGS += -DRTW_USE_CFG80211_STA_EVENT # only enable when kernel >= 3.2
+EXTRA_CFLAGS += -DCONFIG_P2P_IPS
+#EXTRA_CFLAGS += -DCONFIG_DEBUG_RTL871X -DCONFIG_DEBUG
+
+SUBARCH := arm
+ARCH ?= $(SUBARCH)
+CROSS_COMPILE ?=
+KVER  := $(shell uname -r)
+#KSRC := /lib/modules/$(KVER)/build
+MODDESTDIR := /lib/modules/$(KVER)/kernel/drivers/net/wireless/
+INSTALL_PREFIX :=
+endif
+
+
 ifeq ($(CONFIG_PLATFORM_I386_PC), y)
 EXTRA_CFLAGS += -DCONFIG_IOCTL_CFG80211
 EXTRA_CFLAGS += -DRTW_USE_CFG80211_STA_EVENT # only enable when kernel >= 3.2
-- 
2.7.4

