From 4610f54d820562674c63fb48cc2f24b3a0986c00 Mon Sep 17 00:00:00 2001
From: Christophe Priouzeau <christophe.priouzeau@st.com>
Date: Fri, 22 Jul 2016 14:50:44 +0200
Subject: [PATCH 09/12] V4L2-CORE

Signed-off-by: Christophe Priouzeau <christophe.priouzeau@st.com>
---
 drivers/media/v4l2-core/videobuf2-core.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/media/v4l2-core/videobuf2-core.c b/drivers/media/v4l2-core/videobuf2-core.c
index 9fbcb67..99008ba 100644
--- a/drivers/media/v4l2-core/videobuf2-core.c
+++ b/drivers/media/v4l2-core/videobuf2-core.c
@@ -749,6 +749,13 @@ int vb2_core_reqbufs(struct vb2_queue *q, enum vb2_memory memory,
 	if (ret)
 		return ret;
 
+	/* Invalidate request if driver request more than MAX buffers*/
+	if (num_buffers > VIDEO_MAX_FRAME) {
+		dprintk(1, "Driver set the num_buffer (%d) greater than max\n",
+				num_buffers);
+		return -EINVAL;
+	}
+
 	/* Finally, allocate buffers and video memory */
 	allocated_buffers =
 		__vb2_queue_alloc(q, memory, num_buffers, num_planes, plane_sizes);
-- 
1.9.1

